zest/releaser/pypi.py

```diff
@@ -423,30 +423,31 @@ def version_levels(self):
             return default
         return result
 
-    def tag_format(self):
-        """Return the string format for the tag name used in the release.
+    def tag_format(self, version):
+        """Format the tag that should be used in the release.
 
-        Override the default ``%(version)s`` in ~/.pypirc or setup.cfg using
+        Override the default ``{version}`` in ~/.pypirc or setup.cfg using
         a ``tag-format`` option::
 
             [zest.releaser]
-            tag-format = v%(version)s
+            tag-format = v{version}
 
-        Returns default of ``%(version)s`` when nothing has been configured.
```

Comment:
>     -        Returns default of ``%(version)s`` when nothing has been configured.
> This line should stay, right? (just change the `%` for `{` )

I thought the first line of the doc was enough in establishing that `"{version}"` is the default format:

    Override the default ``{version}`` in ~/.pypirc or setup.cfg [...]

I'll reword to call out the default more explicitly.